caluma/caluma_form/schema.py

```diff
@@ -963,3 +933,34 @@ def resolve_all_format_validators(self, info):
 
     def resolve_document_validity(self, info, id):
         return validate_document(info, id)
+
+
+QUESTION_ANSWER_TYPES = {
+    models.Question.TYPE_MULTIPLE_CHOICE: ListAnswer,
+    models.Question.TYPE_INTEGER: IntegerAnswer,
+    models.Question.TYPE_FLOAT: FloatAnswer,
+    models.Question.TYPE_DATE: DateAnswer,
+    models.Question.TYPE_CHOICE: StringAnswer,
+    models.Question.TYPE_TEXTAREA: StringAnswer,
+    models.Question.TYPE_TEXT: StringAnswer,
+    models.Question.TYPE_TABLE: TableAnswer,
+    models.Question.TYPE_FILE: FileAnswer,
+    models.Question.TYPE_DYNAMIC_CHOICE: StringAnswer,
+    models.Question.TYPE_DYNAMIC_MULTIPLE_CHOICE: ListAnswer,
+}
+
+QUESTION_OBJECT_TYPE = {
```

Comment:
Good work on refactoring out those lists! However I'd prefer this being plural named (`QUESTION_OBJECT_TYPES`)